bbb-recording-importer/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.bigbluebutton</groupId>
+    <artifactId>bbb-recording-importer</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.22.2</version>
+            </plugin>
+            <plugin>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <version>2.22.2</version>
+            </plugin>
+        </plugins>
```

Comment:
It would be nice to have a plugin for auto-code formatting, so no one can mess it up:

```xml
<plugin>
<groupId>net.revelc.code.formatter</groupId>
<artifactId>formatter-maven-plugin</artifactId>
<version>2.16.0</version>
<executions>
	<execution>
		<goals>
			<goal>format</goal>
		</goals>
	</execution>
</executions>
<configuration>
	<compilerSource>1.8</compilerSource>
	<compilerCompliance>1.8</compilerCompliance>
	<compilerTargetPlatform>1.8</compilerTargetPlatform>
	<lineEnding>LF</lineEnding>
	<encoding>UTF-8</encoding>
</configuration>
</plugin>
```